### PR TITLE
verify committed *.pb.go files are up-to-date in presubmit checks

### PR DESCRIPTION
**Pull request by ayj**

### CODE REVIEW

**Comment by ZackButcher** [comment]
We should add more OWNERS to /bin at least, and probably to /prow too - two people is too few for as many people as we have working on this project.

File: prow/istio-presubmit.sh [hunk 1]
```diff
@@ -67,6 +67,12 @@ cd $ROOT
 # Build
 ${ROOT}/bin/init.sh
 
+./bin/generate-protos.sh || die "Could not generate *.pb.go"
+if [[ -n $(git status --porcelain) ]]; then
+    git status
+    die "Repo has unstaged changes. Re-run ./scripts/generate-protos.sh"
```
Comment (guptasu) [inline prow/istio-presubmit.sh]: Sweet !!

**Pull request merged**